packages/graphql-codegen-cli/src/loaders/documents/document-loader.ts

```diff
@@ -21,12 +21,15 @@ export const loadFileContent = (filePath: string): DocumentNode | null => {
       return null;
     }
   } else {
-    throw new DetailedError(`
+    throw new DetailedError(
+      'Failed to load a document',
```

Comment:
with that, we gain shorter error messages that are displayed next to the task in listr